umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md

```diff
@@ -0,0 +1,161 @@
+* **Enable the latest version of .NET** for each environment on your Cloud project.
+
+<figure><img src="../../.gitbook/assets/runtime-settings.png" alt=""><figcaption><p>Runtime settings</p></figcaption></figure>
+
+## Step 2: Clone down your environment
+
+* Clone down the **Live** environment.
+* Build and run the [project locally](../set-up/working-locally.md#running-the-site-locally).
+* Log in to the backoffice.
+* Restore content from your Cloud environment.
+
+## Step 3: Upgrade the project locally using Visual Studio
+
+* Open your project in Visual Studio - use the `csproj` file in the `/src/UmbracoProject` folder.
+* Right-click your project solution in **Solution Explorer**.
+* Select **Properties**.
+
+<figure><img src="images/Solution-Explorer.png" alt=""><figcaption></figcaption></figure>
+
+* Select the **.Net 7** **Target Framework** drop-down in the **General** section of the **Application** tab as on your Cloud project.
+
+![Target Framework](images/Target-Framework.png)
+
+* Go to **Tools** > **NuGet Package Manager** > **Manage NuGet Packages for Solution...**.
+* Navigate to the **Browse** tab.
+* Install version 7.0.0 of the `Microsoft.Extensions.DependencyInjection.Abstractions` framework.
+* Navigate to the **Installed** tab.
+* Choose **Umbraco.Cms**.
+* Select your project.
+* Choose **the latest stable version of Umbraco 11** from the **Version** drop-down.
+* Click **Install** to upgrade your project.
+
+![Nuget Version Install](images/Nuget-Version-Install.png)
+
+### Upgrading Add-on packages
+
+Update the following packages to the latest stable version as well:
+
+* Umbraco.Deploy.Cloud
+* Umbraco.Deploy.Contrib
+* Umbraco.Forms
+* Umbraco.Deploy.Forms
+* Umbraco.Cloud.Identity.Cms
+* Umbraco.Cloud.StorageProviders.AzureBlob
+
+{% hint style="info" %}
+Choose the package version corresponding to the CMS version that you are currently upgrading to.
+
+For example, if you are upgrading to "Umbraco.Cms 11.0.0" update the forms package to "Umbraco.Forms 11.0.0" as well.
+
+Also, if you have more projects in your solution or other packages, make sure that these are also updated to support tUmbraco 11.
+{% endhint %}
+
+## Step 4: Finishing the Upgrade
+
+* Enable [Unattended Upgrades](../../umbraco-cms/reference/configuration/unattendedsettings.md#upgrade-unattended)
+* Run the **project locally**
+* Log in to the Umbraco backoffice to **verify the upgrade** has happened.
+* **Remove** Unattended Upgrades
+* **Build and run** the project to verify everything works as expected.
+
+![Target Framework](images/verify-v10-upgrade-locally.png)
+
+Once the Umbraco project runs locally without any errors, the next step is to deploy our Umbraco 10 project to our **clean Umbraco 11 project**.
+
+## Migrate Umbraco 10 project to Umbraco 11 project
+
+The following steps will guide you through the process of deploying your locally upgraded Umbraco 10 project to the **clean Umbraco 11 project**.
```

Comment:
This is very similar to line 97 - consider only saying this once!

Reply:
Fixed it :)